cmd/mysterium_server/command_run/factory.go

```diff
@@ -73,3 +75,15 @@ func NewCommandWith(
 		},
 	}
 }
+
+func SelectIdentity(identityHandler *identityHandler, keyOption string) (id identity.Identity, err error) {
```

Comment:
Idea of handler was, to have thiner code in command.
This why, I moved it to `identity_handler.go` next to `command.go`.

Reply:
well, `SelectIdentity` is independent function - maybe it's worth to move it to new file to make less code in this file?

Reply:
Clear, decoupled from from handler struct.
This separate function could still be in `node_handler.go`

Reply:
Moved it back to `node_handler` as a function.